druid-shell/src/backend/gtk/window.rs

```diff
@@ -907,17 +907,28 @@ impl WindowState {
         for op in queue {
             match op {
                 DeferredOp::Open(options, token) => {
-                    let file_info = dialog::get_file_dialog_path(
+                    // Keep the value of this option for later
+                    let multi_selection = options.multi_selection;
+                    let file_infos = dialog::get_file_dialog_path(
                         self.window.upcast_ref(),
                         FileDialogType::Open,
                         options,
                     )
-                    .ok()
-                    .map(|s| FileInfo {
-                        path: s.into(),
-                        format: None,
+                    // If result is an error, return an empty vector,
+                    // otherwise return a vec of FileInfos
+                    .map_or(vec![], |f| {
```

Comment:
I know this is the same behavior as before, but it does seem like the errors from `get_file_dialog_path` should get logged, at least...

Reply:
Makes sense, I'll add some log here